### ISSUE TITLE
How to use results (e.g. minio or aws s3)

### ISSUE THREAD

**Issue by ForcemCS** [tektoncd/results#788]
As described here [Results](https://tekton.dev/docs/results/),I don't really understand, is there a sample yaml file for reference
`Tekton Results aims to help users logically group CI/CD workload history and separate out long term result storage away from the Pipeline controller. This allows you to:

Provide custom Result metadata about your CI/CD workflows not available in the Tekton TaskRun/PipelineRun CRDs (for example: post-run actions)
Group related workloads together (e.g. bundle related TaskRuns and PipelineRuns into a single unit)
Make long-term result history independent of the Pipeline CRD controller, letting you free up etcd resources for Run execution.
Store logs produced by the TaskRuns/PipelineRuns so that completed Runs can be cleaned to save resources.`

**Comment by enarha** [comment]
@ForcemCS The documentation currently does not include examples for using minio/S3, but you can have a look here https://github.com/openshift-pipelines/pipeline-service/tree/main/operator/gitops/argocd/pipeline-service/tekton-results , specifically https://github.com/openshift-pipelines/pipeline-service/blob/main/operator/gitops/argocd/pipeline-service/tekton-results/api-s3-config.yaml where the S3 configuration is set using env variables (you can also use the api config  https://github.com/openshift-pipelines/pipeline-service/blob/main/operator/gitops/argocd/pipeline-service/tekton-results/config.env to set those). Here is how we create the minio (S3) bucket https://github.com/openshift-pipelines/pipeline-service/tree/main/developer/openshift/gitops/argocd/pipeline-service/tekton-results.
HTH

**Comment by pkking** [comment]
Since tekton result does not ship a CRD, is that means i must expose the tekton-result API server and query server directly to find the result info infomation?

**Comment by enarha** [comment]
Yes. It operates on PipelineRun and TeskRun CRDs prinarily managed by the Tekton Pipeline controller. And yes, one should query the Results APi service for stored information.
